lib/awspec/helper/finder/firehose.rb

```diff
@@ -0,0 +1,13 @@
+module Awspec::Helper
+  module Finder
+    module Firehose
+      def find_delivery_stream(name)
+        res = firehose_client.describe_delivery_stream({
+                                                         delivery_stream_name: name,
+                                                         limit: 1
+                                                       })
+        res.delivery_stream_description
```

Comment:
Coud you use Array#single_resource ( see https://github.com/k1LoW/awspec/blob/master/lib/awspec/ext/array.rb ) and remove `limit`

Example: https://github.com/k1LoW/awspec/blob/a1004610d062c8513e00134913e8f5df5e5c2807/lib/awspec/helper/finder/rds.rb#L14